mongo_connector/connector.py

```diff
@@ -61,6 +60,17 @@ def always(self, message, *args, **kwargs):
     'required': ssl.CERT_REQUIRED
 }
 
+_min_source_version = None
+"""The minimum MongoDB version in the source cluster."""
+
+
+def get_mininum_mongodb_version():
+    return _min_source_version
```

Comment:
Is there a better way to make the source MongoDB version visible in the DocManagers?